pycontw/pycon.tw
# feat(schedule): revamp schedule display in grid

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
Based on the [wireframe](https://docs.google.com/document/d/1sxfUqtaPaaJjnVXBvcazdEXa2oQrUPnZ9VwBoo_jkN4) provided by the design team:
- Display the schedule in grid system to provide users a glimpse of overall events including break time etc.
- Tweak events data in `views.py` to help frontend display the grid layout easier
- Take the liberty temporarily to design the style of the events and the schedule

> There are still some details to better realize the ideal of the wireframe

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to the `/admin` to make sure there are events created just like the ones in the existing Google sheet schedule - including Keynote events, Custom events, Talk events, Tutorial events
2. Go to the `/events/schedule/new` to review the laid out schedule
3. Click on the **Generate** button at the bottom of the `/events/schedule/new`
4. See if the new schedule is published at the `/events/schedule`

## Expected behavior
A clear and concise description of what you expected to happen.

## Related Issue
#813

## More Information
**Screenshots**

- Locate at `/events/schedule`
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/7352613/91064765-452d4680-e662-11ea-8411-106ff1e43110.png">

- Scroll down the schedule timetable
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/7352613/91064836-5ece8e00-e662-11ea-98a4-a09f4d3d5045.png">

- Hover an event (also hover at the long-line title)
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/7352613/91065195-c4227f00-e662-11ea-91ca-e4e4b7f9b36b.png">
